### DISCUSSION TITLE
Split Screens on Web Version

### DISCUSSION THREAD

**Discussion by eagle856** [Ylianst/MeshCentral#4273] [Q&A]
I know you can do split screens for multiple remote monitors with MeshCentral Router. Is this possible with the web version? If not are there any plans to add this feature? I'd really like to give my users the ability to just go to our website to access their computers remotely rather than using the Router (which would probably confuse the hell out them).

**Comment by si458** [comment]
You can already sorta do it...

Connect to desktop and select monitor 1

Then duplicate the tab

Then drag the new tab to another window, 

Then connect again and select monitor 2 

If will show 2 connected

**Reply by eagle856** [reply]
Thanks. I tried that, but it duplicates the screens in both windows. I can click monitor 1 on one screen and it displays monitor 1 on both screens and same thing if I click monitor 2 on either one.

**Comment by DMoRiaM** [comment]
Hi! The only way I could achieve that was by using two different 
browsers. Have you tried that?

Em 14/07/2022 16:36, eagle856 escreveu:
>
> Thanks. I tried that, but it duplicates the screens in both windows. I 
> can click monitor 1 on one screen and it displays monitor 1 on both 
> screens and same thing if I click monitor 2 on either one.
>
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/Ylianst/MeshCentral/discussions/4273#discussioncomment-3149855>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AGDWLMGIUKO4MV3AFIW6QD3VUBT2BANCNFSM53S46VSQ>.
> You are receiving this because you are subscribed to this 
> thread.Message ID: 
> ***@***.***>
>
-- 
Assinatura de E-mail

**Reply by eagle856** [reply]
I'm sure that would work, although I haven't tried it yet. But I feel that may be too difficult for some of our users though (some of them are not the brightest unfortunately). The "Split" button on the router does a great job. I was really hoping @Ylianst might chime in and say there are plans to add it to the web version.
